build.gradle

```diff
@@ -764,6 +767,7 @@ try (BufferedReader inStream = new java.io.BufferedReader(new java.io.InputStrea
             // should the need arise.
 //            ext.jdk18OrLater = jdkVersionInfo.feature() >= 18
 
+            ext.jdk23OrLater = jdkVersionInfo.feature() >= 23
             ext.jdk24OrLater = jdkVersionInfo.feature() >= 24
```

Comment:
We really only need to add `--sun-misc-unsafe-memory-access=allow` when running with JDK 24 or later ("allow" is the default in JDK 23). I want to do initial testing on JDK 23 with this flag, but will likely change all of the checks to `jdk24OrLater` for the final version, in which case we won't need the `jdk23OrLater` flag.

Reply:
I made the above change.